.helm/ecamp3/files/db_backup_job/.env-example

```diff
@@ -0,0 +1,9 @@
+APP_NAME=test-db-backup-job
+DATABASE_URL=
+#if you want to backup your local database:
+#DATABASE_URL=postgresql://ecamp3:ecamp3@<your-ip>:5432/ecamp3dev
```

Comment:
What is `<your-ip>` when developing locally? My IP from which perspective? Would `localhost` work? Do I need to fill in my external IP (I hope the backup isn't performed from outside my network..?)

Reply:
you need to fill your external ip because it runs in a docker container, and i did not add the docker-host-forwarder service to this docker compose